Source/ExcelDna.AddIn.Tasks/CreateExcelAddIn.cs

```diff
@@ -148,6 +150,21 @@ private void TryCreateTlb()
             }
         }
 
+        private void TryPublishDoc()
+        {
+            string docFile = AddInFileName ?? ProjectName + "-AddIn" + ".chm";
```

Comment:
I'm guessing we need some brackets here.

Reply:
Thank you for noticing. I've corrected it.